bundles/org.openhab.binding.modbus.lambda/README.md

```diff
@@ -0,0 +1,174 @@
+# Lambda Heat Pump
+
+This extension adds support for the Lambda Heat Pump modbus protocol as provided by
+https://lambda-wp.at/wp-content/uploads/2022/01/Modbus-Protokoll-und-Beschreibung.pdf 
+
+A Lambda Heat Pump has to be reachable within your network. 
+If you plan to use the E-Manager part to hand over your PV excess to the heat pump ask Lambda support to 
+configure it to  
+E-Meter Kommunikationsart:          ModBus Client
+E-Meter Messpunkt:                  E-Eintrag
+
+Other configurations of the E-Manager are not supported (yet).
+
+Up to now only the following configuration is supported:
+- only one heatpump(heatpump1)
+- only one buffer for the heating(buffer1)
+- only one buffer for the water heating boiler1)
+
+
+## Supported Things
+
+This bundle adds the following thing types to the Modbus binding.
+Note, that the things will show up under the Modbus binding.
+
+| Thing              | ThingTypeID | Description                                         |
+| ------------------ | ----------- | --------------------------------------------------- |
+| Lambda Heat Pump   | lambdahp    | A lambda heat pump visible in the local network     |
```

Comment:
The openHAB model (things, channels etc) is the most important as changing that is breaking and affects users. 
The implementation is less important as it can be changed without breaking (in most cases).
Without looking at the code it would da like you want the shared code to be extracted to a base handler and have the different things extend it.

Reply:
It's not a question if the bundle is working or not. It is about having proper model that is future fit, we like to prevent breaking changes to users.
I can understand that the the stiebeleletron was used as example as it is somehow similar (modbus) and therefore usefull. But it is also a very old binding (way older then 3 years) that originates from openHAB 2.x and maybe even before that. A lot has changed since.

To me there is no reason to rush, i'd rather have a good modelled binding instead of rushing this and have breaking changes later on. I'm happy to assist if you need any help to change this. 

_PS: Please don't mark something as resolved when it is not yet._